components/home/lists/lists-controller.js

```diff
@@ -429,51 +429,66 @@ trackerCapture.controller('ListsController',function(
             }, function(){});
         }
 
-        $scope.useLabTestForProgram = function(program) {
-            return program && program.id == 'B7gOGodZkcs';
+        $scope.isInnreiseProgram = function(program) {
+            return program && program.id == INNREISE_PROGRAM_ID;
         }
 
-        $scope.hasStartedSync = false;
+        $scope.proveSvarSyncIsLoading = false;
         $scope.syncLabTests = function () {
-            if($scope.useLabTestForProgram($scope.selectedProgram)) {
+
+            if($scope.isInnreiseProgram($scope.selectedProgram)) {
+                $scope.provesvarStartFailed = false;
                 var userId;
                 try{
                     userId = JSON.parse(sessionStorage.USER_PROFILE).id
                 }
                 finally {}
-                $scope.hasStartedSync = true;
+                $scope.proveSvarSyncIsLoading = true;
                 FNrLookupService.startLabTestSync($scope.selectedOrgUnit.code, userId).then(function(svar){
-                    if(!svar) {
-                        $scope.hasStartedSync = false;
+                    $scope.proveSvarSyncIsLoading = false;
```

Comment:
Indentation :S

Reply:
Is correct, isn't it?

Reply:
or, the whole block maybe...

Reply:
No, seems correct to me. I'll remove an extra linebreak...

Reply:
You are right :) Got tricked by the lines above